docs/src/explanations/default_outage_values.md

```diff
@@ -0,0 +1,9 @@
+# Default values used for outage statistics
+
+When data is not available in a user-supplied `GeometricDistributionForcedOutage`
+supplemental attribute, then the outage rates default to a set of defaults defined
+in the [Default Outage Rates CSV](https://github.com/NREL-Sienna/SiennaPRASInterface.jl/blob/main/src/util/descriptors/outage-rates-ERCOT-modified.csv) based off of rates in ERCOT.
```

Comment:
Can we make it clear that we only populate default values when no GeometricDistributionForcedOutage objects exist in the System. We don't populate these default values when some components have attributes and others don't. In the case, they will be assigned 0 0 like you mentioned below. is this making sense?

Reply:
Ok, yes that makes sense. Fixed?

Reply:
I think so. I made one final comment on your change.